### PR TITLE
USDZExporter: export from compressed texture data #23321

### PR DESCRIPTION
**Pull request by jeffbeene**
Related issue: #23321

Implement TextureUtils `decompress` function in USDZExporter, allowing export of compressed meshes and textures (e.g. draco + ktx2)

Based on work by @hybridherbst introduced here: https://github.com/mrdoob/three.js/pull/23321/commits/706e9cfdcd9123307f9160993aaf34ac77e2255d

coffeemat_quicklook_ar

### CODE REVIEW

File: examples/files.json [hunk 1]
```diff
@@ -423,6 +423,7 @@
 		"misc_exporter_ply",
 		"misc_exporter_stl",
 		"misc_exporter_usdz",
+    "misc_exporter_usdz_compressed",
```
Comment (Mugen87) [inline examples/files.json:426]: Would you mind reverting this change? A new example isn't required for this feature.

File: examples/jsm/exporters/USDZExporter.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import * as THREE from 'three';
```
Comment (Mugen87) [inline examples/jsm/exporters/USDZExporter.js:1]: Side note: It's not good that `USDZExporter` imports the entire core. Like all other modules, the exporter should only import what is actually required. This should be fixed with another PR.

File: examples/files.json (hunk 1 above)
Comment (jeffbeene) [reply examples/files.json]: Understood, thanks!

File: examples/jsm/exporters/USDZExporter.js (hunk 2 above)
Comment (jeffbeene) [reply examples/jsm/exporters/USDZExporter.js:1]: Agreed, I'll plan on submitting a separate PR once/if this is merged.
